code_quality_checks.yml

```diff
@@ -46,17 +45,15 @@ stages:
               set -ex
               cmake-format --version
               # Perform format checking on all CMake files
-              find . -type f -name CMakeLists.txt -or -name '*.cmake' | xargs cmake-format --check
+              find . -type f -name CMakeLists.txt -or -name '*.cmake' | xargs cmake-format -i
+              git diff --exit-code
```

Comment:
Will this now print the diff if formatting is wrong? Nice!

Reply:
Yes it will